Automattic/wp-calypso
# Update form button position everywhere

#### Changes proposed in this Pull Request

This PR is larger than I usually like to make them but it's because it's making a system wide update for the position of all in-page form buttons. It involves centrally updating the component but then also required some little touch ups across a number of screens. More details here: p9Jlb4-1Rl-p2

**Before**
![image](https://user-images.githubusercontent.com/6981253/97026908-057ec000-1528-11eb-9179-95ba4881541a.png)

**After**
![image](https://user-images.githubusercontent.com/6981253/97026958-17606300-1528-11eb-8abe-c9ea597e37a9.png)

For RTL languages:
![image](https://user-images.githubusercontent.com/6981253/97026803-dec08980-1527-11eb-9744-f1fe2b38cd01.png)


#### Testing instructions
Browse around calypso and confirm that in-page form submission buttons appear on the left side and that there aren't any funky side effects like collapsed margins or text overlapping buttons. This is a list of screens I put together while testing:

- /me
- /me/account
- /me/purchases/:site/:receipt/payment/add
- /me/purchases/:site/:receipt/payment/edit
- /me/purchases/billing/:receipt
- /me/security
- /me/notifications
- /me/privacy
- /me/get-apps
- /import
- /export
- /marketing/sharing-buttons
- /domains/manage/:domain/name-servers/:site
- /domains/manage/:domain/edit-contact-info/:site
- /domains/manage/:domain/dns/:site
- /domains/manage/:domain/transfer/out/:site (for eligible domains)
